### PR TITLE
bug 1889488: [e2e]: kube_pod_resource_requests renamed to kube_pod_container_resource_requests

### PR DESCRIPTION
**Pull request by ingvagabund**
Related: https://github.com/openshift/cluster-kube-scheduler-operator/issues/322

### CODE REVIEW

**Comment by ingvagabund** [comment]
@damemi FYI

File: test/e2e/scheduler_test.go [hunk 1]
```diff
@@ -307,8 +307,8 @@ func TestMetricsAccessible(t *testing.T) {
 		t.Fatalf("error creating route client for prometheus: %v", err)
 	}
 	for _, metric := range []string{
-		"scheduler_schedule_attempts_total", // returned by /metrics
-		"kube_pod_resource_request",         // returned by /metrics/resources
+		"scheduler_schedule_attempts_total",    // returned by /metrics
+		"kube_pod_container_resource_requests", // returned by /metrics/resources
```
Comment (soltysh) [inline test/e2e/scheduler_test.go:311]: /hold 
The change is wrong based on https://github.com/openshift/kubernetes/blob/master/pkg/scheduler/metrics/resources/resources.go#L54

**Comment by smarterclayton** [comment]
Yeah, this is not correct.

**Comment by damemi** [comment]
@ingvagabund was there a resolution to what was wrong with this test? I don't remember when we were talking about it

**Comment by ingvagabund** [comment]
Yeah, just wait for https://github.com/openshift/cluster-kube-scheduler-operator/pull/319 to merge and have the changes propagated to the installer.

**Pull request closed without merging**